app/models/report/summary_collection_builder.rb

```diff
@@ -3,16 +3,24 @@ class Report::SummaryCollectionBuilder
 
   # hash for converting qtypes to groups (stat, select, date, raw) used for generating summaries
   QTYPE_TO_SUMMARY_GROUP = {
-    'integer' => 'stat',
-    'counter' => 'stat',
-    'decimal' => 'stat',
-    'time' => 'stat',
-    'datetime' => 'stat',
-    'select_one' => 'select',
-    'select_multiple' => 'select',
-    'date' => 'date',
-    'text' => 'raw',
-    'long_text' => 'raw'
+    integer: "stat",
+    counter: "stat",
+    decimal: "stat",
+    time: "stat",
+    datetime: "stat",
+    select_one: "select",
+    select_multiple: "select",
+    date: "date",
+    text: "raw",
+    long_text: "raw",
+    barcode: "raw",
+    location: "raw",
+    image: "raw",
```

Comment:
What will image/video/audio summaries look like? Seems this would need more work. Thoughts?

Reply:
@smoyth It looks like it's not even possible to report on image/video/audio columns (those qtypes are not available in the report builder popup).

Reply:
Should I just remove them here?  Or do we want to include them in the report somehow?